packages/types/types/src/record.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export interface IConsumer<T> {
+    /**
+     * Invoked whenever the data this object is bound to is changed.
+     */
+    valueChanged<U extends T, K extends keyof U>(property: K, producer: IProducer<U>): void;
+}
+
+export interface IReader<T> {
+    /**
+     * Return the value associated with `property`.
+     * @param property - The property of the Producer to read.
+     */
+    get<K extends keyof T>(property: K): T[K];
```

Comment:
Previously, this was 'T[K] | Pending<T[K]>'.

I was thinking I would change users of the type to `IReader<T | Pending<T>>`, but nothing complained?

Reply:
Taking out pending from this type is a good change. Users would probably want to use a mapped type over T that changes each property to include pending.

Not sure why there are no complaints. Might be because I moved away from these in examples. Did rush test work? I know that some of the test code is more lax than the source.

Reply:
Yeah, tests passed.  Maybe we only test Pending w/CalcObj?

Reply:
...and thanks...  you're right, it would be the fields (not the record) that are typed as pending.